evergreen/run-tests.sh

```diff
@@ -1,4 +1,4 @@
-#!/bin/bash
+#!/usr/bin/env bash
```

Comment:
This makes is so that the path is searched for bash. This is especially useful on when the user may be using newer version of bash that's not stored at /bin/bash (e.g. homebrew bash on MacOS)